src/igvfd/audit/matrix_file.py

```diff
@@ -11,11 +11,11 @@
 @audit_checker('MatrixFile', frame='object')
 def audit_matrix_file_dimensions(value, system):
     '''
-        audit_detail: Matrix files are expected to have different values for each dimension.
+        audit_detail: Non-hic matrix files are expected to have different values for each dimension.
         audit_category: identical dimensions
         audit_levels: WARNING
     '''
-    if value['dimension1'] == value['dimension2']:
+    if value['dimension1'] == value['dimension2'] and value['file_format'] != 'hic':
         detail = (
             f'Matrix file {audit_link(path_to_text(value["@id"]), value["@id"])} '
             f'is expected to have different values for each dimension, but has '
```

Comment:
maybe instead of "expected to have different values for each dimension," it should be "expected to have different dimensions "?

Reply:
done